src/kv/test/kv_test.cpp

```diff
@@ -15,73 +14,62 @@
 #include <string>
 #include <vector>
 
-struct RawMapTypes
+struct MapTypes
 {
   using StringString = kv::Map<std::string, std::string>;
   using NumNum = kv::Map<size_t, size_t>;
   using NumString = kv::Map<size_t, std::string>;
   using StringNum = kv::Map<std::string, size_t>;
 };
 
-struct ExperimentalMapTypes
-{
-  using StringString = kv::experimental::Map<std::string, std::string>;
-  using NumNum = kv::experimental::Map<size_t, size_t>;
-  using NumString = kv::experimental::Map<size_t, std::string>;
-  using StringNum = kv::experimental::Map<std::string, size_t>;
-};
-
-TEST_CASE_TEMPLATE("Map creation", MapImpl, RawMapTypes, ExperimentalMapTypes)
+TEST_CASE("Map creation")
```

Comment:
This file is just a regex replace: `TEST_CASE_TEMPLATE(XXX, ...)` becomes `TEST_CASE(XXX)`.